libr/cons/cons.c

```diff
@@ -697,6 +697,19 @@ R_API int r_cons_get_cursor(int *rows) {
 	return col;
 }
 
+R_API bool r_cons_isatty() {
+#if __UNIX__ || __CYGWIN__
+	struct winsize win = { 0 };
+        if (isatty (1) && !ioctl (0, TIOCGWINSZ, &win)) {
+                if ((win.ws_col == 0) || (win.ws_row == 0)) {
```

Comment:
It doesn't work even with !=0 && !=0 because r_file_exists only returns true for regular files (not character devices).